docs/content/users/quickstart.md

```diff
@@ -375,6 +375,31 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
 
     For more advanced tasks like adding elasticsearch, building and watching storefront and administration, see [susi.dev](https://susi.dev/ddev-shopware-6).
 
+=== "Silverstripe"
+
+    ## Silverstripe
+
+    === "Silverstripe"
+
+        Use a new or existing Composer project, or clone a Git repository.
+
+        === "Composer"
+            ```bash
+            mkdir my-silverstripe-app
+            cd my-silverstripe-app
+            ddev config --project-type=silverstripe --docroot=public --create-docroot
+            ddev composer create --prefer-dist --no-scripts silverstripe/installer -y
+            ddev start
+            ```
+        === "Git Clone"
+            ```bash
+            git clone <your-silverstripe-repo>
+            cd <your-silverstripe-project>
+            ddev config --project-type=silverstripe --docroot=public --create-docroot
+            ddev start
+            ddev composer install
```

Comment:
I would suggest to add a `ddev launch /admin` at the end of the `composer` and `git clone` snippets which would be in line with the other quickstarter guides.

Reply:
added a note about it.

I'm not a fan of including it inside the command list, as it's not relevant to the actual set-up itself.